### PR TITLE
Newsweek.com Homepage Clone Finished!

### PR DESCRIPTION
**Pull request by SlowKingV**
> This project requests students to build a replica of the news site [Newsweek.com](https://www.newsweek.com) using the Bootstrap framework.

## :tada: MILESTONES :tada:
- [x] Use **Semantic HTML** to build the page structure.
- [x] Add CDN version of ***[Bootstrap 4](https://getbootstrap.com/)*** into your project.
- [x] Use **CSS** to give proper styles to the content.
- [x] Use ***[Bootstrap 4](https://getbootstrap.com/)*** to align everything properly and make the page fully responsive!!!
- [x] Use **[W3C Validator](validator.w3.org/)** to find and fix any error inside the *index.html* file.
- [x] Set up a *linter* into the repo and solve any problem it might find.

***Everything is ready, waiting for approval   :thumbsup:***

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1117 @@
...
+                    <div class="latest-news d-flex flex-column pl-md-3">
+
+                        <article class="d-flex border-bottom py-2 py-md-0 align-items-md-start">
+                            <figure class="flex-grow-0 flex-shrink-0 mr-3 mb-0"><a href="#"><img src="assets/img/article/other/stray-kids-k-pop-boy-band-seoul.jpg" alt="placeholder"></a></figure>
+                            <h3 class="mb-0 pb-md-3"><a href="">#Levanter1stWin Trends As K-pop Group Stray Kids Marks First 'Levanter' Win</a></h3>
+                        </article>
+
+                        <article class="d-flex border-bottom py-2 py-md-0 align-items-md-start">
+                            <figure class="flex-grow-0 flex-shrink-0 mr-3 mb-0 pt-3"><a href="#"><img src="assets/img/article/other/twitch.jpg" alt="placeholder"></a></figure>
+                            <h3 class="mb-0 py-md-3"><a href="">Canadian Mother Accused of Assaultin
```
Comment (pelzolga123) [inline index.html]: It's better to delete comments.

Comment (pelzolga123) [inline index.html]: It's better to delete comments.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Bootstrap should be added using links.

File: index.html (hunk 1 above)
Comment (pelzolga123) [inline index.html]: It's better to delete comments.

**Comment by pelzolga123** [comment]
## Hi Diego!  
### Your project looks :+1: amazing! You did very good job!
### Kudos for most descriptive README file I've seen so far :100: 

I see only one thin that needs correction:
* [x] Html should reference Bootstrap CSS file like [this website](https://getbootstrap.com/docs/4.3/getting-started/introduction/#css) shows. Right now you just add it manually. [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_requirement.png)  

### Please kindly fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by SlowKingV** [comment]
### I got rid of all the local Bootstrap files and replaced the links for the ones on the page :+1:

**Comment by izaiasneto4** [comment]
### Hi @SlowKingV :wave: 

Nice job implementing the requested changes! Project approved! :+1: 

TSE: Izaias Neto

**Pull request merged**